src/jit/emitxarch.cpp

```diff
@@ -10724,7 +10774,6 @@ BYTE* emitter::emitOutputRI(BYTE* dst, instrDesc* id)
         // Handle SSE2 instructions of the form "opcode reg, immed8"
 
         assert(id->idGCref() == GCT_NONE);
-        assert(valInByte);
```

Comment:
Why did this assert get removed?

Reply:
Because `valInByte` would be true only if imm-atgument is in 0-127.

Reply:
Commented here: https://github.com/dotnet/coreclr/pull/16183#discussion_r170110807, we can continue the conversation there to prevent two threads 😄